css.css

```diff
@@ -0,0 +1,128 @@
+body {
+  background: rgb(188,86,188);
+background: radial-gradient(circle, rgba(188,86,188,1) 0%, rgba(150,182,255,1) 100%);
+}
+.container {
+  width: 1200px;
+  height: 100%;
+  margin: auto;
+  padding-top: 30px;
+}
+
+/* Search */
+.search {
+
+}
+
+.search h2 {
+  margin: 0;
+  margin-bottom: 10px;
+}
+
+
+
+/* Tests */
+#TestsAndResult {
+
+}
+
+.wrapper {
+  display: flex;
+}
+
+/* Arguments */
+#tests {
+  margin-right: 40px;
+}
+
+#testsList {
+  padding-left: 3px;
+  direction:rtl;
```

Comment:
Please test this properly.

Reply:
this is there to make scrollbars appear on the left(for whatever perverted reason), and works fine since the elements inside this container set direction back to ltr